arm-trafficmanager/2015-11-01/trafficmanager.json

```diff
@@ -0,0 +1,699 @@
+            "type": "string",
+            "description": "The name of the Traffic Manager endpoint to be deleted."
+          },
+          {
+            "name": "endpointType",
+            "in": "path",
+            "required": true,
+            "type": "string",
+            "description": "The type of the Traffic Manager endpoint to be deleted."
+          },
+          {
+            "name": "endpointName",
+            "in": "path",
+            "required": true,
+            "type": "string",
+            "description": "The name of the Traffic Manager endpoint to be deleted."
+          },
+          {
+            "$ref": "#/parameters/ApiVersionParameter"
+          },
+          {
+            "$ref": "#/parameters/SubscriptionIdParameter"
+          }
+        ],
+        "responses": {
+          "200": { 
+             "description": "", 
+             "schema": { 
+               "$ref": "#/definitions/Endpoint" 
+             } 
+          } 
+        }
+      }
+    },
+    "/providers/Microsoft.Network/checkTrafficManagerNameAvailability": {
+      "post": {
+        "tags": [
+          "Profiles"
+        ],
+        "operationId": "Profiles_CheckTrafficManagerRelativeDnsNameAvailability",
+        "description": "Create or update a Traffic Manager endpoint.",
+        "parameters": [
+          {
+            "name": "parameters",
+            "in": "body",
+            "required": true,
+            "schema": {
+              "$ref": "#/definitions/CheckTrafficManagerRelativeDnsNameAvailabilityParameters"
+            },
+            "description": "The Traffic Manager name parameters supplied to the CheckTrafficManagerNameAvailability operation."
+          },
+          {
+            "$ref": "#/parameters/ApiVersionParameter"
+          },
+          {
+            "$ref": "#/parameters/SubscriptionIdParameter"
+          }
+        ],
+        "responses": {
+          "200": { 
+             "description": ""
+          } 
+        }
+      }
+    },
+    "/subscriptions/{subscriptionId}/resourceGroups/{resourceGroupName}/providers/Microsoft.Network/trafficmanagerprofiles": {
+      "get": {
+        "tags": [
+          "Profiles"
+        ],
+        "x-ms-pageable": {
+          "nextLinkName": null
+        },
+        "operationId": "Profiles_ListAllInResourceGroup",
+        "description": "Lists all Traffic Manager profiles within a resource group.",
+        "parameters": [
+          {
+            "name": "resourceGroupName",
+            "in": "path",
+            "required": true,
+            "type": "string",
+            "description": "The name of the resource group containing the Traffic Manager profiles to be listed."
+          },
+          {
+            "$ref": "#/parameters/ApiVersionParameter"
+          },
+          {
+            "$ref": "#/parameters/SubscriptionIdParameter"
+          }
+        ],
+        "responses": {
+          "200": { 
+             "description": ""
```

Comment:
Please make sure that description is present and is correct every where. This description will become the documentation of your clientside library. There is an effort going on by another team to create REST API docs for every service using the swagger spec. Hence, it is of utmost importance to have the description accurate.